internal/pkg/deploy/cloudformation/project.go

```diff
@@ -0,0 +1,269 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package cloudformation provides functionality to deploy archer resources with AWS CloudFormation.
```

Comment:
nit: we can remove the package comment from this file, it only needs to be in one of them.

Reply:
👍